packages/wallet-core/src/store/actions/sendTransaction.ts

```diff
@@ -43,15 +44,20 @@ export const sendTransaction = async (
   const _asset = assetsAdapter(asset)[0];
   const _feeAsset = assetsAdapter(feeAsset)[0] || _asset;
 
-  const tx = await client.wallet.sendTransaction({
-    to: getChain(network, chainId).formatAddress(to),
-    value: new BN(amount),
-    data,
-    gasLimit: gas,
-    fee,
-    asset: _asset,
-    feeAsset: _feeAsset,
-  });
+  const parser = getErrorParser(ChainifyErrorParser);
+  const tx = (await parser.wrapAsync(
```

Comment:
Is there any way to make the wrapping of chainify calls seamless? 
It strikes me as verbose and error prone to have to wrap every call individually. 
Instead you could return proxies that handle chainify errors when client is created in `clients.ts`

Reply:
Done